joomla/joomla-cms
# Remove cacerts bundled in Joomla and replace with composer package

### Summary of Changes
Replaces the outdated cacerts bundled with Joomla HTTP and FOF Download with the latest versions (from 2015 and 2017 respectively)

### Testing Instructions
Ensure that the HTTP Package and Downloads package continue to function and use the updated CA Trust

### Documentation Changes Required
None
